azurerm/internal/services/kusto/tests/resource_arm_kusto_cluster_test.go

```diff
@@ -30,6 +30,35 @@ func TestAccAzureRMKustoCluster_basic(t *testing.T) {
 	})
 }
 
+func TestAccAzureRMKustoCluster_propsUpdate(t *testing.T) {
```

Comment:
Could we just use 
```suggestion
func TestAccAzureRMKustoCluster_update(t *testing.T) {
```

for consistency